kolibri/plugins/device/assets/src/views/ManageContentPage/index.vue

```diff
@@ -229,9 +229,14 @@
         }[value];
         this.$router.push(this.$router.getRoute(nextRoute));
       },
-      showNewLabel(channelId) {
+      showNewLabel(channelId, channelVersion) {
         const match = find(this.installedChannelsWithResources, { id: channelId });
-        return match && match.taskIndex > -1;
+        // Check if channel match exists and is a newer version or has resources
+        return (
+          match &&
+          (match.version > channelVersion ||
```

Comment:
Oh wait - is this logic the correct way around? `channelVersion` is the version of the channel on the remote, right? So we'd want to flag it in the case that `match.version < channelVersion`?